WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleCardPresentPaymentInLineMessage.swift

```diff
@@ -33,6 +36,14 @@ struct PointOfSaleCardPresentPaymentInLineMessage: View {
             PointOfSaleCardPresentPaymentCancelledOnReaderMessageView(viewModel: viewModel)
         }
     }
+
+    // MARK: - Animations
+
+    /// Used together with .matchedGeometryEffect
+    /// This makes SwiftUI treat different messages as a single view in the context of animation.
+    /// Allows to smoothly transition from one view to another while also transitioning to full-screen
+    @Namespace private var transitionAnimation
+    private static let transitionAnimationId = UUID().uuidString
```

Comment:
Since this is a static let, there's probably no need for it to be so unique as a UUID... 

Only reason I raise it is in case this string could in future be surfaced in a console or warning or error, it would be easier to debug with something human readable, if we can get away with that?

Reply:
Yes, we can use a normal string as well 👍